bin/test.ps1

```diff
@@ -1,2 +1,5 @@
+#Requires -Modules @{ ModuleName = 'Pester'; MaximumVersion = '4.99' }
+
 if(!$env:SCOOP_HOME) { $env:SCOOP_HOME = Resolve-Path (scoop prefix scoop) }
```

Comment:
```suggestion
if(!$env:SCOOP_HOME) { $env:SCOOP_HOME = Convert-Path (scoop prefix scoop) }
```

Reply:
> Resolves the wildcard characters in a path, and displays the path contents.

See [Resolve-Path](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/resolve-path?view=powershell-5.1)

For most usage, we should use `Convert-Path` which just `Converts a path from a PowerShell path to a PowerShell provider path.` unless using `Resolve-Path -Relative`.

Reply:
Forgotten to change them in `Main` bucket...